src/main/resources/MiLoG_Template.tex

```diff
@@ -3,241 +3,252 @@
 \usepackage{tabulary}
```

Comment:
\usepackage{tabulary} should be deleted because the document does not use tabulary anymore.

Reply:
Yes, removed it.